apps/console/src/home.jsp

```diff
@@ -23,6 +23,7 @@
 <jsp:scriptlet>
     session.setAttribute("authCode", request.getParameter("code"));
     session.setAttribute("sessionState", request.getParameter("session_state"));
+    if(request.getParameter("state") != null) {session.setAttribute("state", request.getParameter("state"));}
```

Comment:
```suggestion
    if (request.getParameter("state") != null) {
        session.setAttribute("state", request.getParameter("state"));
    }
```

Reply:
This is intentionally made inline, as this is inside a `<jsp:scriptlet>` tag